src/mygrad/nnet/activations/glu.py

```diff
@@ -0,0 +1,72 @@
+from numpy import ndarray
+
+from mygrad import multiply, Tensor
+from .sigmoid import sigmoid
+
+
+def glu(x, dim=-1, constant=False):
```

Comment:
I think this should be `axis` to stick with the parlance of NumPy

Reply:
:flushed: why did I do that? :facepalm: updated